src/resources/PackageResource.ts

```diff
@@ -49,7 +51,7 @@ export class PackageResource extends Resource {
     }
 
     const response = await request.fetch();
-    return (await response.json()) as Package[];
+    return (await response.json()) as Responses.BulkGetResponse<Package>;
```

Comment:
```suggestion
    return response.json()
```

Since the function returns a promise, do we need to resolve the promise here with `await`? I would think the caller of this function would handle the async response.

Reply:
I thought that we'll need to type-cast the resolved promise to another type. But it turns out returning the promise directly works! 🙌